docs/src/content/docs/components/icons.mdx

```diff
@@ -0,0 +1,132 @@
+---
+title: Icons
+description: Learn how to display icons in Starlight.
+---
+
+import { Icon } from '@astrojs/starlight/components';
+
+To display icons from a set of [common icons](/reference/icons/#all-icons), use the `<Icon>` component.
```

Comment:
Couple of nits here:
- you can’t use “a set” — there’s only one supported set currently, so we should use a definite article (or like in this case specify “our” icons
- I think rather than “common”, which in English often means “widespread” or “frequent”, maybe something like “built-in” is a better way to describe these (mirrors the component language too “built-in components” / “built-in icons”)

```suggestion
To display icons from Starlight’s [built-in icon set](/reference/icons/#all-icons), use the `<Icon>` component.
```